documentation/develop/firefox-workflow-overview.md

```diff
@@ -34,12 +34,12 @@ date: 2019-05-20 03:25:40
 {% capture content %}
 
 <table>
-    <tr>
-        <th style="width:20%"><img src="{% asset "documentation/develop/firefox_workflow_prepare.png" @path @optim %}"></th>
-        <th style="width:20%"><img src="{% asset "documentation/develop/firefox_workflow_code.png" @path @optim %}"></th>
-        <th style="width:20%"><img src="{% asset "documentation/develop/firefox_workflow_publish.png" @path @optim %}"></th>
-        <th style="width:20%"><img src="{% asset "documentation/develop/firefox_workflow_enhance.png" @path @optim %}"></th>
-        <th style="width:20%"><img src="{% asset "documentation/develop/firefox_workflow_retire.png" @path @optim %}"></th>
+    <tr align="left">
+        <th><h3>Prepare</h3></th>
```

Comment:
This might need styling without it, but it might be nicer to leave the `h3` out of this.